### ISSUE TITLE
Release 0.4.2

### ISSUE THREAD

**Issue by skevy** [skevy/graphiql-app#32]
Waiting on the following PR's to merge into GraphiQL (presumably causing a release of graphiql@0.7.3).

- [x] https://github.com/graphql/graphiql/pull/144
- [x] https://github.com/graphql/graphiql/pull/145

**Comment by gjtorikian** [comment]
Both https://github.com/graphql/graphiql/pull/144 and https://github.com/graphql/graphiql/pull/143 have been merged, so I think that gets us going for a `0.4.2`.

**Comment by kevinold** [comment]
@skevy I pushed up #39 and can merge and do a release if you give the 👍 .  Let me know.

**Comment by skevy** [comment]
Sorry I didn't see this @kevinold @gjtorikian! Updating now

**Comment by skevy** [comment]
Released!
